README.md

```diff
@@ -398,59 +431,59 @@ exit /b 0
 popd
 echo Failed.
 exit /b 1
+```
+
+# Licensing
+
+## License of Minetest Source Code
+
+##### Minetest
+Copyright (C) 2010-2017 celeron55, Perttu Ahola <celeron55@gmail.com>
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-License of Minetest textures and sounds
----------------------------------------
+## License of Minetest Textures and Sounds
 
 This applies to textures and sounds contained in the main Minetest
 distribution.
 
 Attribution-ShareAlike 3.0 Unported (CC BY-SA 3.0)
 http://creativecommons.org/licenses/by-sa/3.0/
 
-Authors of media files
------------------------
+## Authors of Media Files
+
 Everything not listed in here:
 Copyright (C) 2010-2012 celeron55, Perttu Ahola <celeron55@gmail.com>
 
-ShadowNinja:
+##### ShadowNinja:
   textures/base/pack/smoke_puff.png
 
-Paramat:
+##### Paramat:
   textures/base/pack/menu_header.png
 
-erlehmann:
+#####erlehmann:
   misc/minetest-icon-24x24.png
   misc/minetest-icon.ico
   misc/minetest.svg
   textures/base/pack/logo.png
 
-License of Minetest source code
--------------------------------
-
-Minetest
-Copyright (C) 2010-2017 celeron55, Perttu Ahola <celeron55@gmail.com>
-
-This program is free software; you can redistribute it and/or modify
-it under the terms of the GNU Lesser General Public License as published by
-the Free Software Foundation; either version 2.1 of the License, or
-(at your option) any later version.
-
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU Lesser General Public License for more details.
-
-You should have received a copy of the GNU Lesser General Public License along
-with this program; if not, write to the Free Software Foundation, Inc.,
-51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
-
-Irrlicht
----------------
+## Irrlicht
 
-This program uses the Irrlicht Engine. http://irrlicht.sourceforge.net/
+This program uses the [Irrlicht Engine.](http://irrlicht.sourceforge.net/)
```

Comment:
Period should be outside of the link text:

    This program uses the [Irrlicht Engine](http://irrlicht.sourceforge.net/).